### ISSUE TITLE
LoadImage - TypeError: Object of type Image is not JSON serializable

### ISSUE THREAD

**Issue by ghostsquad** [Chaoses-Ib/ComfyScript#50]
How to use LoadImage? I'm getting this error 

```
TypeError: Object of type Image is not JSON serializable
```

specifically, I'm trying to add this to my workflow (some hardcoded values for now):

```py
def cnet_zoedepth(ctx, img):
    control_net = ControlNetLoaderAdvanced(ControlNetLoaderAdvanced.control_net_name.diffusers_xl_depth_mid, None)
    img = ImageResize(img, 'rescale', 'true', 'lanczos', 1, ctx.width, ctx.height)
    preprocessed_img = ZoeDepthAnythingPreprocessor(img, 'indoor', ctx.width)
    ctx.pos, ctx.neg, ctx.model = ACNAdvancedControlNetApply(ctx.pos, ctx.neg, control_net, preprocessed_img, strength=0.4, start_percent=0, end_percent=0.4)
```

```py
img, _ = LoadImage("D:\sd\input\clipspace\clipspace-rgba-534537233.png")
cnet_zoedepth(ctx, img)
```

**Comment by Chaoses-Ib** [comment]
It works on my side. Maybe the error isn't caused by these code?

![image](https://github.com/Chaoses-Ib/ComfyScript/assets/50246090/db9a61a1-eb4d-482b-b11e-f3f8c03a43ea)

![image](https://github.com/Chaoses-Ib/ComfyScript/assets/50246090/2ac368ec-7357-4911-9882-c9b4b5be65d0)

**Comment by ghostsquad** [comment]
hmmm, I think this was user error. I separated out some functions into a separate block in the notebook. Probably just forgot to rerun that.

**Comment by ghostsquad** [comment]
Took a break from SD in general. Coming back to this. I'm using image loader like this:

```Python
image = LoadImage(
      image=LoadImage.image._20250216_060809_pass1_0001_png
    )
```

and encountering this error:

TypeError: Object of type Image is not JSON serializable

This _feels_ like the right way to use this, as I get autocompletion, but maybe not, do I need to resort to manually writing the path?

**Comment by ghostsquad** [comment]
nope, once again... stupid Python, stupid error messages, this was because some function returned a tuple, but I didn't assign that tuple to enough variables. UGHHH

**Comment by Chaoses-Ib** [comment]
This is actually a common mistake. Maybe I should add a check for tuple 🤔, like if there is serialization error and the value is a tuple, tell the user to have a check
